tox.ini

```diff
@@ -1,26 +1,42 @@
 [tox]
-# rmeoved docs, pypy, pypy3, py{36,37}-pytest{master,features}
-envlist=linting,py{36,37,38},docs
+envlist = py{36,37,38}-{linux,macos,windows},docs,napari,linting
 toxworkdir=/tmp/.tox
 
+[gh-actions]
+python =
+    3.6: py36
+    3.7: py37
+    3.8: py38
+
+[gh-actions:env]
+PLATFORM =
+    ubuntu-latest: linux
+    macos-latest: macos
+    windows-latest: windows
+
 [testenv]
-commands=
-  {env:_PLUGGY_TOX_CMD:pytest} {posargs}
-  coverage: coverage report -m --omit="*testing*"
-  coverage: coverage xml --omit="*testing*"
-setenv=
-  _PYTEST_SETUP_SKIP_PLUGGY_DEP=1
-  coverage: _PLUGGY_TOX_CMD=coverage run -m pytest
-extras=testing
-deps=
-  coverage: coverage
+platform = linux: linux
+           macos: darwin
+           windows: win32
+extras=test
+commands = pytest --color=yes -v --cov --cov-report=xml {posargs}
+
 
 [testenv:linting]
 skip_install = true
-basepython = python3.6
 deps = pre-commit
 commands = pre-commit run --all-files --show-diff-on-failure
 
+[testenv:napari]
+basepython = python3.8
+deps = 
+    -rhttps://raw.githubusercontent.com/napari/napari/master/requirements/test.txt
```

Comment:
Can we put a space after the `r`? I thought this was a typo for a second

Reply:
this seems to be how they[ document it](https://tox.readthedocs.io/en/latest/example/basic.html#depending-on-requirements-txt-or-defining-constraints)... but I just tried with a space and it also works.  weird.